source/DailyOverview/DailyStyles.css

```diff
@@ -50,14 +50,6 @@ h2 {
     display: block;
 }
 
-h1 > a:first-child {
-    text-decoration: none;
-}
-
-h1 > a:first-child:hover {

```

Comment:
I feel like having this text decoration would be useful to the user to make it more clear that this is actually interactive